theSrc/scripts/PlotData.coffee

```diff
@@ -445,7 +444,7 @@ class PlotData
     }
     @outsidePlotPtsId.push id
     @normalizeData()
-    @getPtsAndLabs()
+    @getPtsAndLabs('addElemFromLegend')
```

Comment:
Is this a way to comment on how the code enters the function? Or was this part of some previous debug code?